pkg/common-controller/snapshot_controller.go

```diff
@@ -457,6 +460,9 @@ func (ctrl *csiSnapshotCommonController) syncUnreadySnapshot(snapshot *crdv1.Vol
 					ctrl.updateSnapshotErrorStatusWithEvent(snapshot, v1.EventTypeWarning, "SnapshotStatusUpdateFailed", fmt.Sprintf("Snapshot status update failed, %v", err))
 					return err
 				}
+
+				msg := fmt.Sprintf("waiting for a snapshot %s to be created", snapshot.Name)
+				ctrl.eventRecorder.Event(snapshot, v1.EventTypeNormal, "syncUnreadySnapshot", msg)
```

Comment:
s/syncUnreadySnapshot/CreateSnapshot

Reply:
The reason is "s/syncUnreadySnapshot/CreatingSnapshot" ?